pkg/docker/hub/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM debian:8
+
+# ---------------------------------------------------------------------------
+# Environment
+# ---------------------------------------------------------------------------
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV GOSU_VERSION 1.9
+ENV PIPELINEDB_VERSION 0.9.5
+ENV PIPELINEDB_URL "https://www.pipelinedb.com/download/${PIPELINEDB_VERSION}/debian8"
+ENV PIPELINEDB_TMP_DIR /tmp/pipelinedb
+ENV PIPELINEDB_DEB_FILE ${PIPELINEDB_TMP_DIR}/pipelinedb.deb
+ENV PIPELINEDB_DATA /var/lib/pipelinedb/data
+
+# ---------------------------------------------------------------------------
+# Locales
+# ---------------------------------------------------------------------------
+
+RUN apt-get update \
+    && apt-get install -y locales \
+    && rm -rf /var/lib/apt/lists/* \
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+
+# ---------------------------------------------------------------------------
+# Gosu
+# ---------------------------------------------------------------------------
+
+RUN set -x \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates wget \
+    && rm -rf /var/lib/apt/lists/* \
+    && dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')" \
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" \
+    && wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc" \
+    && export GNUPGHOME="$(mktemp -d)" \
+    && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
+    && gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu \
+    && rm -r "${GNUPGHOME}" /usr/local/bin/gosu.asc \
+    && chmod +x /usr/local/bin/gosu \
+    && gosu nobody true \
+    && apt-get purge -y --auto-remove ca-certificates wget
+
+# ---------------------------------------------------------------------------
+# PipelineDB
+# ---------------------------------------------------------------------------
+
+RUN mkdir -p ${PIPELINEDB_TMP_DIR} \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates wget \
+    && wget -O ${PIPELINEDB_DEB_FILE} ${PIPELINEDB_URL} \
+    && dpkg --install ${PIPELINEDB_DEB_FILE} \
+    && rm -r ${PIPELINEDB_TMP_DIR} \
+    && apt-get purge -y --auto-remove ca-certificates wget
+
+# ---------------------------------------------------------------------------
+# Configuration
+# ---------------------------------------------------------------------------
+
+VOLUME /var/lib/pipelinedb/data
+EXPOSE 5432
+
+RUN addgroup --system pipeline \
+    && adduser \
+        --system \
+        --no-create-home \
+        --disabled-login \
+        --ingroup pipeline \
+        pipeline
+
+COPY docker-entrypoint.sh /usr/local/bin/
```

Comment:
Ah i see oops. It's gitignored somewhere. Sry I haven't seen that. I'll push in a sec.